config/loader_test.go

```diff
@@ -0,0 +1,151 @@
+package config_test
+
+import (
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/odpf/optimus/config"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	configFileName       = ".optimus.yaml"
+	optimusConfigDirName = "./optimus"
+	optimusConfigContent = `
```

Comment:
nit: Create a separate const for this one alone.

Reply:
updated with separating it from the other two